src/core/tests/type_prop/segment_max.cpp

```diff
@@ -14,16 +14,8 @@
 #include "openvino/op/shape_of.hpp"
 #include "openvino/op/strided_slice.hpp"
 
-namespace ov {
-namespace test {
-
-using ov::op::v0::Constant;
-using ov::op::v0::Parameter;
-using ov::op::v1::Add;
-using ov::op::v1::ReduceMax;
-using ov::op::v1::StridedSlice;
-using ov::op::v3::ShapeOf;
-using testing::HasSubstr;
```

Comment:
Restore this `using testing::HasSubstr;` and  adding `testing::` in test will be not requried.